### PR TITLE
fix(Scripts/Underbog): Rework Swamplord Muselek

### PR DESCRIPTION
**Pull request by Nyeriah**
## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Gultask — APPROVED** [review]

File: src/server/scripts/Outland/CoilfangReservoir/underbog/boss_swamplord_muselek.cpp [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                        }, 3s);
+                    }
+                });
+            }
+
+            context.Repeat(12s, 16s);
+        });
+    }
+
+    void UpdateAI(uint32 diff) override
```
Comment (acidmanifesto) [inline src/server/scripts/Outland/CoilfangReservoir/underbog/boss_swamplord_muselek.cpp:164]: Diff unused

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Comment by Kitzunu** [comment]
You forget to load the new script file :)

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/scripts/Outland/outland_script_loader.cpp

**Comment by Nyeriah** [comment]
Ye ye, this is not ready for review...
I still need to change a lot of stuff, I just reopened it cus the other one got screwed by merge conflicts that I cba to solve

**Comment by Maelthyrr** [comment]
Always in WIP?

**Comment by Nyeriah** [comment]
Need to script the bear and I can’t manage to get his movement to work to my hearts content as of yet, more important stuff required my attention so this got put in the freezer

**Pull request merged**